Locations.py

```diff
@@ -1,4 +1,7 @@
 from BaseClasses import Location, MultiWorld
+
+from MetroidPrimeInterface import MetroidPrimeLevel

```

Comment:
This required a relative import when I ran it locally
```suggestion
from .MetroidPrimeInterface import MetroidPrimeLevel

```